vector/src/main/java/im/vector/app/features/media/BaseAttachmentProvider.kt

```diff
@@ -120,24 +117,28 @@ abstract class BaseAttachmentProvider(val imageContentRenderer: ImageContentRend
             }
         })
 
-        target.onVideoFileLoading(info.uid)
-        fileService.downloadFile(
-                downloadMode = FileService.DownloadMode.FOR_INTERNAL_USE,
-                id = data.eventId,
-                mimeType = data.mimeType,
-                elementToDecrypt = data.elementToDecrypt,
-                fileName = data.filename,
-                url = data.url,
-                callback = object : MatrixCallback<File> {
-                    override fun onSuccess(data: File) {
-                        target.onVideoFileReady(info.uid, data)
-                    }
-
-                    override fun onFailure(failure: Throwable) {
-                        target.onVideoFileLoadFailed(info.uid)
+        if (data.url?.startsWith("content://").orFalse() && data.allowNonMxcUrls) {
```

Comment:
Maybe if you write `== true` instead of `.orFalse()` you will not be forced to use `!!` in the next line (I have not check that)